src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/AutomatedPanningMachine.java

```diff
@@ -62,38 +62,41 @@ public AutomatedPanningMachine(ItemGroup itemGroup, SlimefunItemStack item) {
     @Override
     public void onInteract(Player p, Block b) {
         ItemStack input = p.getInventory().getItemInMainHand();
+        Material material = input.getType();
+        ItemStack output;
```

Comment:
```suggestion
        ItemStack output = null;
```
for constitency we should set this to null